app/models/supervisor.rb

```diff
@@ -37,6 +37,14 @@ def pending_volunteers
     ).where(invitation_accepted_at: nil).where.not(invitation_created_at: nil)
   end
 
+  def inactive_volunteers
+    recent_case_contact_volunteer_ids = volunteers.joins(:case_contacts).where(
+      case_contacts: {created_at: 30.days.ago..}
```

Comment:
Changed `occurred_at` to `updated_at`. Since these can be done in the past, a volunteer could create a contact 5 days in the past, and then the occurred_at might be misleading in the future.